### PR TITLE
FIX Use Updated names of top page extensions

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/1240

### CODE REVIEW

File: src/Extensions/UsedOnTableExtension.php [hunk 1]
```diff
@@ -23,7 +24,11 @@ protected function updateUsageAncestorDataObjects(array &$ancestorDataObjects, D
         if (!class_exists(BaseElement::class) || !is_a($owner, BaseElement::class)) {
             return;
         }
-        $page = $owner->hasExtension(TopPageExtension::class) ? $owner->getTopPage() : $owner->getPage();
+        if ($owner->hasExtension(TopPageElementExtension::class) || $owner->hasExtension(TopPageFluentElementExtension::class)) {
+            $page = $owner->getTopPage();
+        } else {
+            $page = $owner->getPage();
+        }
```
Comment (ChloeSartorelli) [inline src/Extensions/UsedOnTableExtension.php:31]: I prefer the ternary... but it's not worth ping pong.

Comment (ChloeSartorelli) [reply src/Extensions/UsedOnTableExtension.php:31]: But the change to logic isn't needed as discussed irl. Please remove the unnecessary addition

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by emteknetnz** [comment]
Have validated locally that hasExtension() will perform an is_a() check that will check for subclasses. Don't need this PR

**Comment by emteknetnz** [comment]
Actually we do :laughing: renamed the class, re-opening, and have removed the check for the fluent class

**Comment by emteknetnz** [comment]
Wouldn't let me reopen as I had force pushed over the last commit, and opened this instead https://github.com/silverstripe/silverstripe-linkfield/pull/327

**Pull request closed without merging**